File: DesktopApplication/DataObjects/Question.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataObjects
{
    public class Question
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public Theme Theme { get; set; }
        public Boolean Choice { get; set; }

        public Question(int id, string title, string description, Theme theme, Boolean choice)
        {
            this.Id = id;
            this.Title = title;
            this.Description = description;
            this.Theme = theme;
            this.Choice = choice;
        }
    }
}

File: JavaDesktopApp/src/be/pxl/classes/Question.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package be.pxl.classes;


/**
 *
 * @author Jordy
 */
public class Question {
    private int ID;
    private String Title;
    private String Description;
    
    private Theme Theme;
    
    
    private boolean Choice;
    
    public Question(int ID, String Title, String Description, Theme Theme, boolean Choice) {
    this.ID = ID;
    this.Title = Title;
    this.Description = Description;
    this.Theme = Theme;
    this.Choice = Choice;
    }

    /**
     * @return the ID
     */
    public int getID() {
        return ID;
    }

    /**
     * @param ID the ID to set
     */
    public void setID(int ID) {
        this.ID = ID;
    }

    /**
     * @return the Title
     */
    public String getTitle() {
        return Title;
    }

    /**
     * @param Title the Title to set
     */
    public void setTitle(String Title) {
        this.Title = Title;
    }

    /**
     * @return the Description
     */
    public String getDescription() {
        return Description;
    }

    /**
     * @param Description the Description to set
     */
    public void setDescription(String Description) {
        this.Description = Description;
    }

    /**
     * @return the Choice
     */
    public boolean isChoice() {
        return Choice;
    }

    /**
     * @param Choice the Choice to set
     */
    public void setChoice(boolean Choice) {
        this.Choice = Choice;
    }

    /**
     * @return the Theme
     */
    public Theme getTheme() {
        return Theme;
    }

    /**
     * @param Theme the Theme to set
     */
    public void setTheme(Theme Theme) {
        this.Theme = Theme;
    }
    
}
